page_v2.go

```diff
@@ -122,6 +122,10 @@ func (dp *dataPageReaderV2) read(r io.Reader, ph *parquet.PageHeader, codec parq
 		}
 	}
 
+	if !ph.DataPageHeaderV2.IsCompressed {
```

Comment:
Wow. I guess we can create an issue in github and ask them why don't they do this. Or even better create a PR to handle this

Reply:
Yup, that's the next step ;-) 
I'll import this into our own prod first though, to battle test it.